specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -4888,7 +4915,24 @@
       ],
       "x-ms-enum": {
         "name": "IndexerStatus",
-        "modelAsString": false
+        "modelAsString": false,
+        "values": [
+          {
+            "value": "unknown",
+            "name": "Unknown",
+            "description": "Indicates that the indexer is in an unknown state."
+          },
+          {
+            "value": "error",
+            "name": "Error",
+            "description": "Indicates that the indexer experienced an error that cannot be corrected without human intervention."
```

Comment:
Nit: maybe elide "human". Could be automated. Just seems like an over-qualification.

*I am not a bot*